frontend/isoko/src/pages/ListBusiness.tsx

```diff
@@ -178,24 +182,75 @@ const StyledCheck = styled(Form.Check)`
 const HourLabel = styled(Form.Label)`
    min-width: 70px;
 `;
-const ListBusiness: React.FC = () => {
-   const [err, setErr] = useState('');
 
-   const [isOwner, setIsOwner] = useState('');
-   const [isBrickAndMortar, setIsBrickAndMortar] = useState('');
-
-   const [businessName, setBusinessName] = useState('');
-   const [street, setStreet] = useState('');
-   const [city, setCity] = useState('');
-   const [state, setState] = useState('');
-   const [zip, setZip] = useState('');
-   const [tags, setTags] = useState([]);
-   const [category, setCategory] = useState('');
-   const [businessURL, setBusinessURL] = useState('');
-   const [shortDesc, setShortDesc] = useState('');
-   const [ownerName, setOwnerName] = useState('');
-   const [ownerPhone, setOwnerPhone] = useState('');
-   const [ownerDesc, setOwnerDesc] = useState('');
+export enum Day {
+   Mon = 'Mon',
+   Tue = 'Tue',
+   Wed = 'Wed',
+   Thu = 'Thu',
+   Fri = 'Fri',
+   Sat = 'Sat',
+   Sun = 'Sun',
+}
+
+interface Hours {
+   [Day.Mon]?: string;
+   [Day.Tue]?: string;
+   [Day.Wed]?: string;
+   [Day.Thu]?: string;
+   [Day.Fri]?: string;
+   [Day.Sat]?: string;
+   [Day.Sun]?: string;
+}
+
+interface Link {
+   title: string;
+   link: string;
+}
+
+interface ListBusinessProps extends React.HTMLProps<HTMLDivElement> {
+   name: string;
+   city: string;
+   category: string;
+   state: string;
+   street: string;
+   zip: string;
+   ownerName: string;
+   ownerPhone: string;
+   ownerDesc: string;
+   description: string;
+   stars: number;
+   minorityTags: string[];
+   keywordTags: string[];
+   verified: boolean;
+   numReviews: number;
+   hours: Hours;
+   links: Link[];
+}
+
+const ListBusiness: React.FC<ListBusinessProps> = (props) => {
+   const [err, setErr] = useState('');
+   const [isOwner, setIsOwner] = useState(props.ownerDesc ? 'true' : '');
```

Comment:
in order to even be on this page they have to be the owner, right? so probably no need to use state here, maybe just set the box to be permanently checked and always have the owner desc visible

Reply:
The updateInfo form is the same component as the listBusiness form, so it won't always be accessed in the context of updateInfo. The isOwner would not false for a user who is listing a business on behalf of the owner.